botnim/kb/manager.py

```diff
@@ -0,0 +1,179 @@
+from pathlib import Path
+import requests
+import io
+from typing import List, Union, BinaryIO, Tuple
+from .base import VectorStore
+from ..config import get_logger
+
+logger = get_logger(__name__)
+
+class ContextManager:
+    def __init__(self, config_dir: Path, kb_backend: VectorStore):
+        self.config_dir = config_dir
+        self.kb_backend = kb_backend
+
+    def _add_environment_suffix(self, name: str) -> str:
+        """Add environment suffix if not in production"""
+        if not self.kb_backend.production:
+            name_parts = name.rsplit('.', 1)
+            return f"{name_parts[0]} - פיתוח.{name_parts[1]}" if len(name_parts) > 1 else f"{name} - פיתוח"
+        return name
+
+    def create_vector_store(self, context_config: dict) -> str:
```

Comment:
probably should be delegated to the kb_backend implementation to decide how many vector stores it supports (one or multiple)

Reply:
done.